### DISCUSSION TITLE
Reverse scrolling doesn't appear to work when trying to scroll through Applications from the Dock

### DISCUSSION THREAD

**Discussion by TheRoyalScam** [Caldis/Mos#637] [Q&A]
Hello, I'm coming from Mac Mouse Fix to Mos (after reading good things, and because it has features I need). Love the smooth scrolling, but I'm noticing an anomaly on Reverse Scrolling.

It works on every app I've tried, and every Finder window. But when I click on the Applications folder in the Dock (my settings are: Display as Folder; View Content as Grid) and try to scroll in the window that pops up from the Dock, Reverse Scrolling is not working. That wasn't the case with Mac Mouse Fix. Is this a bug in Mos that I should report?

(I'm on an Intel MacBook Pro 2019 i7, running macOS Ventura 13.6.4)

**Comment by strbyd** [comment]
I'm also facing this issue. I'm on MacBook Pro 2024 running macOS Sequoia 15.6. I wanted to replace Scroll Reverser with MOS as an all-in-one solution but this bug is really annoying
